PYME/LMVis/sessionpaths.py

```diff
@@ -120,16 +121,16 @@ def substitute_sessiondir(session_txt, session_filename):
     ----------
     session_txt : str
         session dictionary as a string
-    sessiondir : str
-        path to the session directory (i.e. the directory containing the session file)
+    session_filename : str
+        path to the session file
 
     Returns
     -------
     str
         session_txt with SESSIONDIR_TOKEN replaced by sessiondir
     """
     from pathlib import Path
-    sessiondir = Path(session_filename).resolve().parent
+    sessiondir = Path(session_filename).resolve().parent.as_posix()
```

Comment:
This is the crucial fix, otherwise we get an error from replace as it cannot deal with the path object